libs/cardano-data/cardano-data.cabal

```
@@ -1,6 +1,6 @@
 cabal-version:      3.0
 name:               cardano-data
-version:            1.0.0.0
+version:            1.1.0.0
```

This is not a breaking change, thus only a minor version bump is required:
```suggestion
version:            1.0.1.0
```

We also want to add a lower bound in mary, just to be on the safe side: `cardano-data >= 1.0.1`